### PR TITLE
Feature/threading

### PR DESCRIPTION
**Pull request by DavidClawson**
Completed work on parseDecorators.

### CODE REVIEW

File: Aries/Aries/Source/Connections/Messages/SignedConnectionResponseMessage.swift [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//
+//  SignedConnectionResponseMessage.swift
+//  Aries
+//
+//  Created by David Clawson on 10/25/21.
+//
```
Comment (TheTreek) [inline Aries/Aries/Source/Connections/Messages/SignedConnectionResponseMessage.swift:6]: This file should probably be deleted?

File: Aries/Aries/Source/Messaging/MessageReceiver.swift [hunk 2]
```diff
@@ -45,40 +44,117 @@ public class MessageReceiver{
...
+                    case .failure(let error):
+                        print("\(error.localizedDescription)")
+                    case .success(let parsedMessage):
+                        print("Parsed String:   >>>> \(parsedMessage)")
+                        let typeContainer = try! decoder.decode(TypeContainerMessage.self, from: parsedMessage.message.data(using: .utf8)!)
+                        let type = typeContainer.type
+                        print(parsedMessage.message)
+                        self.triggerEvent(type: type, payload: parsedMessage.message.data(using: .utf8)!, senderVerkey: parsedMessage.senderVerkey)
+                    }
+                    return String()
```
Comment (TheTreek) [inline Aries/Aries/Source/Messaging/MessageReceiver.swift:62]: Remove the return of an empty string
```suggestion
                   
```

File: Aries/Aries/Source/Messaging/MessageReceiver.swift [hunk 3]
```diff
@@ -45,40 +44,117 @@ public class MessageReceiver{
...
+                }
             }
         }
+    }
+    
+    private func parseDecorators(message: String, completion: @escaping (_ string: Result<IndyUnpackedMessage, Error>)->String) {
```
Comment (TheTreek) [inline Aries/Aries/Source/Messaging/MessageReceiver.swift:68]: Callback should return void
```suggestion
    private func parseDecorators(message: String, completion: @escaping (_ string: Result<IndyUnpackedMessage, Error>)->Void) {
```

**Review by TheTreek — CHANGES_REQUESTED** [review]
Requested some changes

File: Aries/Aries/Source/Messaging/MessageReceiver.swift (hunk 2 above)
Comment (DavidClawson) [reply Aries/Aries/Source/Messaging/MessageReceiver.swift]: This is necessary with the completion type? I don't fully understand it, but if you remove it, the compiler complains.

**Review by TheTreek — APPROVED** [review]

**Pull request merged**